tasks/main.yml

```diff
@@ -10,3 +10,41 @@
   package:
     name: cockpit-session-recording
   when: install_session_player
+
+- name: configure basic sssd
+  template:
+    src: 'etc/sssd/conf.d/sssd-session-recording.conf.j2'
+    dest: /etc/sssd/conf.d/sssd-session-recording.conf
+    owner: root
+    group: root
+    mode: 0600
+
+- name: configure sssd session recording config
+  template:
+    src: 'etc/sssd/sssd.conf.j2'
+    dest: /etc/sssd/sssd.conf
```

Comment:
We should only create a sssd.conf if one doesn't already exist.  You can do this by providing the `force: no` parameter to the template module.  This parameter defaults to `yes`, which means existing config will be overwritten with your current patch.

Reply:
I also believe that we only need to create `/etc/sssd/sssd.conf` on systems that use older versions of SSSD, as current versions will load config snippets from `/etc/sssd/conf.d` even if `/etc/sssd/sssd.conf` doesn't exist.  We should make this task conditional, perhaps based on the value returned by `sssd --version`.  I have an inquiry out to the SSSD developers to find out what version first added this capability.

Reply:
The version of SSSD where this snippet issue will be fixed is 2.1.

Reply:
As I understand we still must provide basic sssd config in order to session recording to work at all. I am not sure about this though. Anyway we could add this to conf.d later

Reply:
We will not need the basic SSSD config for our session recording snippet to work in SSSD 2.1 and later (this is not released anywhere yet). We do need it now though.  I have confirmed this with the SSSD developers.

Reply:
Right, there was a but that sssd.conf was required, but will just the session recording config through sssd will be enought for sssd to function at all? As I understood it should work, right?